src/js/util.js

```diff
@@ -10,7 +10,7 @@ const Selectors = {
 const ClassName = {
   ACTIVE: 'active',
   LINEAR: 'linear',
-  BLOCK: 'dstepper-block',
+  BLOCK: 'show',
```

Comment:
Please revert this change, I don't want my stepper to depend on Bootstrap, I want to be able to use it without Bootstrap